prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -68,463 +79,399 @@ export EXTERNAL_SOLUTION_TEST_RELEASE_NAME="${EXTERNAL_SOLUTION_TEST_NAMESPACE}"
+# shellcheck source=prow/scripts/lib/gcloud.sh
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/gcloud.sh"
+
+# shellcheck source=prow/scripts/lib/testing-helpers.sh
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/testing-helpers.sh"
 
-# shellcheck disable=SC1090
-source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/library.sh"
-# shellcheck disable=SC1090
-source "${KYMA_SCRIPTS_DIR}/testing-common.sh"
-# shellcheck disable=SC1090
-source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/log.sh"
-# shellcheck disable=SC1090
+# shellcheck source=prow/scripts/cluster-integration/helpers/kyma-cli.sh
 source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/cluster-integration/helpers/kyma-cli.sh"
 
 cleanup() {
-    ## Save status of failed script execution
-    EXIT_STATUS=$?
+  ## Save status of failed script execution
+  EXIT_STATUS=$?
 
-    if [[ "${ERROR_LOGGING_GUARD}" = "true" ]]; then
-        shout "AN ERROR OCCURED! Take a look at preceding log entries."
-        echo
-    fi
+  if [[ "${ERROR_LOGGING_GUARD}" = "true" ]]; then
+    log::error "AN ERROR OCCURED! Take a look at preceding log entries."
+  fi
 
-    #Turn off exit-on-error so that next step is executed even if previous one fails.
-    set +e
+  #Turn off exit-on-error so that next step is executed even if previous one fails.
+  set +e
 
-    # collect logs from failed tests before deprovisioning
-    runTestLogCollector
+  log::banner "CLEANUP"
 
-    if [[ -n "${CLEANUP_CLUSTER}" ]]; then
-        shout "Deprovision cluster: \"${CLUSTER_NAME}\""
-        date
+  # collect logs from failed tests before deprovisioning
+  runTestLogCollector
 
-        #save disk names while the cluster still exists to remove them later
-        DISKS=$(kubectl get pvc --all-namespaces -o jsonpath="{.items[*].spec.volumeName}" | xargs -n1 echo)
-        export DISKS
+  if [[ -n "${CLEANUP_CLUSTER}" ]]; then
+    log::info "Deprovision cluster: \"${CLUSTER_NAME}\""
 
-        #Delete cluster
-        "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/deprovision-gke-cluster.sh"
+    #save disk names while the cluster still exists to remove them later
+    DISKS=$(kubectl get pvc --all-namespaces -o jsonpath="{.items[*].spec.volumeName}" | xargs -n1 echo)
+    export DISKS
 
-        #Delete orphaned disks
-        shout "Delete orphaned PVC disks..."
-        date
-        "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/delete-disks.sh"
-    fi
+    #Delete cluster
+    "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/deprovision-gke-cluster.sh"
 
-    if [[ -n "${CLEANUP_GATEWAY_DNS_RECORD}" ]]; then
-        shout "Delete Gateway DNS Record"
-        date
-        "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/delete-dns-record.sh --project="${CLOUDSDK_CORE_PROJECT}" --zone="${CLOUDSDK_DNS_ZONE_NAME}" --name="${GATEWAY_DNS_FULL_NAME}" --address="${GATEWAY_IP_ADDRESS}" --dryRun=false
-    fi
+    #Delete orphaned disks
+    log::info "Delete orphaned PVC disks..."
+    "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/delete-disks.sh"
```

Comment:
I don't like that the `delete-disks.sh` script takes non visible parameter `DISKS`, Maybe we should also refactor this script to get list of disks as a input parameters?

Reply:
i totally agree with you, but is this really something that needs to be done in this PR? I will either add it to an existing refactoring ticket, or create a new ticket for such suggestions.

Reply:
I would create seperate PR/tiket for this change :)